### ISSUE TITLE
UI colourblindness checks

### ISSUE THREAD

**Issue by lauracumming** [ocadotechnology/codeforlife-design#27]
Review colours within game UI using appropriate tool such as [WCAG] or plug in (https://snook.ca/technical/colour_contrast/colour.html#fg=33FF33,bg=333333). 

Both the games and portal need a review. 
They should be assessed using the recommended standards and tested for the most common types of colourblindness (red/Green)

**Comment by lauracumming** [comment]
This was done
